### PR TITLE
Add ability to turn hints off

### PR DESCRIPTION
**Pull request by canadianveggie**
For those that don't want the temptation.
Also lets us reclaim the vertical real estate on the hard setting.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -568,11 +591,12 @@ <h2>Sharing theme</h2>
     // eslint-disable-next-line no-unused-vars
     function gameState() {
       const DEFAULT_SETTINGS = {
-        keyboardLayout: "atoz", // atoz, qwerty
```
Comment (canadianveggie) [inline index.html:571]: Sorted

**Review by wachunga — COMMENTED** [review]
Was this requested somewhere? eg for classroom projection?

Wondering a little if we have too many options or if it would be useful to group them somehow.

**Comment by canadianveggie** [comment]
> Was this requested somewhere? eg for classroom projection?
> 
> Wondering a little if we have too many options or if it would be useful to group them somehow.

It wasn't requested but thought it would be useful.

**Review by wachunga — APPROVED** [review]
I'm not sure we need it but OK 😄

**Pull request merged**